src/popup/router/components/FungibleTokens/TokensListItem.vue

```diff
@@ -1,11 +1,13 @@
 <template>
-  <RouterLink
+  <Component
+    :is="preventNavigation ? 'div' : 'RouterLink'"
     v-if="tokenData"
     class="tokens-list-item"
     :to="{
```

Comment:
This should be null if the preventNavigation is true to avoid attaching meaningless properties to divs.